src/main/kotlin/com/terraformation/backend/customer/db/ProjectStore.kt

```diff
@@ -151,4 +223,12 @@ class ProjectStore(
               addedBy = currentUser().userId, participantId = participantId, projectId = projectId))
     }
   }
+
+  private fun fetchOrgIdByProjectId(projectId: ProjectId): OrganizationId {
+    return dslContext
+        .select(PROJECTS.ORGANIZATION_ID)
+        .from(PROJECTS)
+        .where(PROJECTS.ID.eq(projectId))
+        .fetchOne(PROJECTS.ORGANIZATION_ID) ?: throw ProjectNotFoundException(projectId)
+  }
```

Comment:
`ParentStore` already has a method for this (and, despite my comment about stores calling stores, `ParentStore` is safe to call from here since it only depends on `DSLContext`.)